pyhole/core/logger.py

```diff
@@ -30,22 +70,21 @@ def setup_logger(name):
 
     debug = utils.debug_enabled()
 
-    log_dir = utils.get_directory("logs")
     log_level = logging.DEBUG if debug else logging.INFO
```

Comment:
This could also be changed to a constant: `LOG_LEVEL = logging.DEBUG if utils.debug_enabled() else logging.INFO`